labelbox/client.py

```diff
@@ -899,6 +907,30 @@ def create_ontology_from_feature_schemas(self,
         normalized = {'tools': tools, 'classifications': classifications}
         return self.create_ontology(name, normalized, media_type)
 
+    def delete_unused_feature_schema(self, feature_schema_id):
+        """
+        Deletes a feature schema if it is not used by any ontologies or annotations
+
+        Args:
+            feature_schema_id (str): The id of the feature schema to delete
+
+        Returns:
+            True if the feature schema was deleted
+        """
+
+        endpoint = self.rest_endpoint + "/feature-schemas/" + feature_schema_id
+        response = requests.delete(
```

Comment:
Nit: It would be good to create a function to handle all types of REST requests, similar to how it's done in SDK 4 Session https://github.com/Labelbox/labelbox-python/blob/sdk4-experimental/labelbox_dev/session.py#L83
since 3 of the PRs make REST requests, but you can feel free to do it in a separate ticket or as a polish item